Documentation/ceph-teardown.md

```diff
@@ -39,10 +39,9 @@ Verify that the cluster CRD has been deleted before continuing to the next step.
 kubectl -n rook-ceph get cluster.ceph.rook.io
 ```
 
-## Delete the Operator and Agent
-This will begin the process of all cluster resources being cleaned up, after which you can delete the rest of the deployment with the following:
+## Delete the Operator
+This will begin the process of all cluster resources being cleaned up, after which you can delete the operator and related resources such as the agent and discover daemonsets with the following:
 ```console
-kubectl delete -n rook-ceph-system daemonset rook-ceph-agent
 kubectl delete -f operator.yaml
```

Comment:
just deleting operator.yaml will also delete the agent/discover daemonsets?

Reply:
yes, the operator.yaml defines the namespace, so they will be cleaned up with the namespace